### DISCUSSION TITLE
Buy PCB

### DISCUSSION THREAD

**Discussion by bluesball** [josefadamcik/SofleKeyboard#189] [Q&A] [answered]
Hi.

I'm newbie in the customization keyboard build
Where is it recommended to buy pcb? Any website? What to look for when purchasing a kit?

**Comment by rjrobsonjunior** [comment, marked as answer]
https://jlcpcb.com/

Minimum order of five pcbs

**Reply by bluesball** [reply]
Tks

**Comment by Khaled-Alsan** [comment]
Simple work can be given to [JLC](https://jlcpcb.com/), cheap and fast! 
Complicated work can be entrusted to [PEAK](https://www.pcbapeak.com/), the minimum order quantity is 1pcs!
I'm using both of these at the moment and they work well, so I hope that helps!
